src/apps/Calculators/AxByEqualsP/AxByReadout.js

```diff
@@ -0,0 +1,105 @@
+import LTUniquesDisplay from "../LTShared/LTUniquesDisplay";
+import { AxByAllModes } from "./AxByAllModes";
+import { handleAxByChange } from "./axByEventHandlers";
+import getAxByScale from "./util/getAxByScale";
+import getSecondCents from "./util/getSecondCents";
+
+export const AxByReadout = ({ stateData, setStateData }) => {
+  const firstSteps = stateData.axByEqualsP.firstSteps;
+  const firstCents = stateData.axByEqualsP.firstCents;
+  const secondSteps = stateData.axByEqualsP.secondSteps;
+
+  const isValidState = () => {
+    return firstSteps > 0 && firstCents > 0 && firstSteps * firstCents < 1200;
+  };
+  if (!isValidState()) {
+    return <InvalidState firstSteps={firstSteps} firstCents={firstCents} />;
+  }
+
+  const secondCents = getSecondCents(firstSteps, firstCents, secondSteps);
+  let scale = [];
+  if (secondCents > 0) {
+    scale = getAxByScale(firstSteps, firstCents, secondSteps, secondCents);
+  }
+  const sclData = [firstSteps, firstCents, secondSteps, secondCents];
+
+  return (
+    <div>
+      <br />
+      <div className="tableClone">
+        <p>
+          Second Interval Steps:{" "}
+          <input
+            onChange={(event) =>
+              handleAxByChange(event, stateData, setStateData)
+            }
+            type="number"
+            name="axBy2StepsEntryField"
+            value={secondSteps}
+          ></input>
```

Comment:
Tag should self-close:

```suggestion
          />
```